src/common/stringutils.cpp

```diff
@@ -24,6 +24,8 @@
 #include <cstdio>
 #include <vector>
 
+#include <cstring>          // memset used for trace
+#include "common/logger.h"
```

Comment:
I reenabled our old linter and it complains about include order here, local includes should go before global ones